manifest.json

```diff
@@ -5,5 +5,14 @@
     "version": "0.0.1",
     "action": {
       "default_popup": "popup/popup.html"
+    },
+    "permissions": ["activeTab", "scripting"],
```

Comment:
Why do we need to add these permissions?

Reply:
Honestly, I just read the google developer article and did what they did. But got it, I'll start working on the functionality stuff that you assigned me.

Reply:
When people are installing the extension, they may get alerts that they need to approve access to search history, webpage content, etc. if we have permissions defined that we don't actually need. We will need some permissions but we will add them as we work on tasks